### ISSUE TITLE
Preview for one Instagram Reel shows as a still image in Discord & Telegram

### ISSUE THREAD

**Issue by superrpotion** [Wikidepia/InstaFix#58]
Link in question: 
https://www.ddinstagram.com/reel/C0rMISPt4wC/

What it looks like on Telegram: 
![IMG_6072](https://github.com/Wikidepia/InstaFix/assets/139422879/feffb6bb-cb6d-4907-bc6b-4be4c6c06bf6)

What it looks like on Discord: 
![IMG_6071](https://github.com/Wikidepia/InstaFix/assets/139422879/348303de-0701-4600-aa15-bc98ce1f81c3)

Is there any way to “refresh” the preview so it shows the video properly? Thank you!

**Comment by superrpotion** [comment]
Found another link with the same issue: 
https://www.ddinstagram.com/reel/C1Jk3FPPXoV/

**Comment by Codingale** [comment]
Friend shared this URL with the same issue:
https://www.instagram.com/reel/C1JYnZ6uqxu/

Maybe all `/reel/` posts have issues right now, as that one simply says post not found.

**Comment by superrpotion** [comment]
> Friend shared this URL with a _similar_ issue: https://www.instagram.com/reel/C1JYnZ6uqxu/
> 
> Maybe all `/reel/` posts have issues right now, as that one simply says post not found.

I don’t believe all reels have the issue, I’ve managed to send quite a few reels with working previews. 

Here’s some examples of reels that generate a video preview properly: 
https://www.ddinstagram.com/reel/CyoFGOfL_iK/
https://www.ddinstagram.com/reel/C1PIhH1vD0X/
https://www.ddinstagram.com/reel/Cy3R4XGr_9u/
https://www.ddinstagram.com/reel/C1HzkxgumvT/

Maybe there’s something that all the videos who can’t generate a preview have in common? Not sure what though. 

.
.

But while we’re on the topic, I found this reel which did not generate a media preview, only the text in the caption. 
Link: https://www.ddinstagram.com/reel/C1SR4A2ulGH/

Telegram preview:
![image](https://github.com/Wikidepia/InstaFix/assets/139422879/49e36a7d-30f8-4e85-9e3b-e1fbd683afc2)

**Comment by Wikidepia** [comment]
> Link in question:
https://www.ddinstagram.com/reel/C0rMISPt4wC/

Yes this is known issue, currently InstaFix is blocked from accessing certain reels, so it will only show the thumbnails. (This is what blocked looks like: https://www.instagram.com/reel/C0rMISPt4wC/embed)

> But while we’re on the topic, I found this reel which did not generate a media preview, only the text in the caption.
Link: https://www.ddinstagram.com/reel/C1SR4A2ulGH/

Sometimes video bigger than 20MB cannot be embed, this is limitation from Telegram.

**Comment by superrpotion** [comment]
> > Link in question:
> > https://www.ddinstagram.com/reel/C0rMISPt4wC/
> 
> Yes this is known issue, currently InstaFix is blocked from accessing certain reels, so it will only show the thumbnails. (This is what blocked looks like: https://www.instagram.com/reel/C0rMISPt4wC/embed)
> 

Thank you for the explanation. So these two links are also the result of Instagram blocking InstaFix from viewing the embed? 
https://www.ddinstagram.com/reel/C1f9YUlPx2U/
https://www.ddinstagram.com/reel/C01MK3FPS63/
